src/components/Button/Button.tsx

```diff
@@ -4,6 +4,7 @@ import Image from 'next/image';
 type ButtonType = 'primary' | 'outline';
 
 export default function Button({
+  ariaLabel = null,
```

Comment:
```suggestion
  ariaLabel = '',
```
changing this to an empty string should fix the lint error in the build